### DISCUSSION TITLE
Azure setup

### DISCUSSION THREAD

**Discussion by pollygen** [Portfolio-Shop/portfolioshop#100] [Q&A] [answered]
Hi! I am unable to set up the azure development settings. Is there any documentation available for this repository?

**Comment by sudip-mondal-2002** [comment, marked as answer]
Hey @pollygen, please head over to the [Contribution.md](https://github.com/Portfolio-Shop/portfolioshop/blob/master/CONTRIBUTING.md) file. We have documented it there. If you have any further queries, please continue on this thread. Thank you.
